app/models/lookup_key.rb

```diff
@@ -162,6 +162,11 @@ def contains_erb?(value)
     value =~ /<%.*%>/
   end
 
+  def overridden?(host)
+    return false unless host.is_a?(Host::Base) || host.is_a?(Hostgroup)
+    lookup_values.find_by_match(host.send(:lookup_value_match)).present?
```

Comment:
I think it should be private... this method reaches into host's internals (which I kind of hate, but I see no choice), thus, using send is okay IMHO...